context/spec/022-unified-design-system/design-system/project/ui_kits/app/kit.css

```
@@ -0,0 +1,131 @@
+/* SavePoint UI Kit — component styles. Mirrors the codebase's shadcn + token
+   classes. Pairs with tokens.css (the design-system foundations). */
+
+* { box-sizing: border-box; }
+html, body { margin: 0; padding: 0; }
+body {
+  background: var(--background);
+  color: var(--foreground);
+  font-family: var(--font-runtime-sans);
+  -webkit-font-smoothing: antialiased;
+  text-rendering: optimizeLegibility;
```

Stylelint (17.12.0)

[error] Expected "optimizeLegibility" to be "optimizelegibility" (value-keyword-case)

(value-keyword-case)